js/components/forms/utils.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Useful methods to interact with redux-form
+ *
+ * @flow
+ */
+
+import * as React from 'react'
+import { type FieldProps } from 'redux-form'
+import { Item, Input, View, Text } from 'native-base'
+
+import I18n from '../../i18n'
+
+const getValidatorMessage = (validatorId: string): string => {
+  return I18n.t(`forms.validators.${validatorId}`)
+}
+
+const required = (value: string): ?string =>
+  value ? undefined : getValidatorMessage('required')
+
+const email = (value: string): ?string =>
+  value && !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(value)
+    ? getValidatorMessage('email')
+    : undefined
+
+/**
+ * A utility function that return the translated value for a property of a form field.
+ */
+export const getFormFieldProperty = (
+  formId: string
+): (string => string => string) => (fieldId: string): (string => string) => (
+  propertyId: string
+): string => {
+  return I18n.t(`forms.${formId}.fields.${fieldId}.${propertyId}`)
+}
+
+/**
+ * Methods used to validate redux-form `Field` components.
+ */
+export const validators = {
+  required,
+  email
+}
+
+/**
+ * This method is used by redux-form `Field` components.
+ * It takes as input the field properties and return a native-base `Input`.
+ */
+export const renderNBInput = ({
```

Comment:
getNativeInput or toNativeInput ? (does this method actually render something or just return the component tree ?)
anyway I'd avoid 'NB' infix which is hard to grasp :)

Reply:
The method name prefix ('render') is the standard used in redux-form https://redux-form.com/7.3.0/examples/fieldlevelvalidation/. 'NB' is for 'NativeBas'e i think is better to replace it with renderNativeBaseInput to be more explicit.